test/service/test_request_builder.py

```diff
@@ -437,6 +438,28 @@ def test_project_summaries(self):
                 "_source": {
                     "entity_id": "b",
                     "contents": {
+                        "samples": [
+                            {
+                                "biomaterial_id": [
+                                    "specimen1"
+                                ],
+                                "disease": [
+                                    "disease1"
+                                ],
+                                "entity_type": "specimens",
+                                "effective_organ": "organ1"
```

Comment:
Shouldn't the specimen have a corresponding `organ` field?

Reply:
Added `organ` property to `specimens` in the test data

Reply:
Then the sample object should also contain the `organ` field, no?